lib/stat_tracker.rb

```diff
@@ -32,4 +32,68 @@ def highest_total_score
   def lowest_total_score
     @game_csv.map { |row| row[:away_goals].to_i + row[:home_goals].to_i }.min
   end
-end
\ No newline at end of file
+
+  def percentage_home_wins
+    home_wins = 0
+    total_wins = 0
+    @game_csv.each do |row|
+      home_wins += 1 if row[:home_goals].to_f > row[:away_goals].to_f
+      total_wins += 1
+    end
+    return (home_wins.to_f/total_wins.to_f).round(2)
+  end
+
+  def percentage_visitor_wins
+    visitor_wins = 0
+    total_wins = 0
+    @game_csv.each do |row|
+      visitor_wins += 1 if row[:home_goals].to_f < row[:away_goals].to_f
+      total_wins += 1
+    end
+    return (visitor_wins.to_f/total_wins.to_f).round(2)
+  end
+
+  def percentage_ties
```

Comment:
These could also potentially fit into the same helper method during the refactor?